serverless/aws/lambda/src/main/scala/sttp/tapir/serverless/aws/lambda/package.scala

```diff
@@ -0,0 +1,7 @@
+package sttp.tapir.serverless.aws
+
+import cats.data.Kleisli
+
+package object lambda {
+  type Route[F[_]] = Kleisli[F, AwsRequest, AwsResponse]
```

Comment:
If we use `AwsRequest => F[AwsResposne]`, can we avoid the dependency on cats?

Reply:
then we would still have dependency to cats in `F: Sync`

Reply:
Ah in the interpreter, right ... but then we could reuse the same `Route` alias for other interpreters as we add them (future/zio based)